tools/vagrant/custom/README.md

```diff
@@ -0,0 +1,176 @@
+## Custom setup for OpenWhisk using Vagrant
+
+The following instructions were tested on Mac OS X El Capitan, Ubuntu 14.04.3 LTS and may work on Windows.
+
+*Requirements*
+- Install [Vagrant](https://www.vagrantup.com/downloads.html)
+
+### Configure Vagrant vm
+
+```
+git clone https://github.com/openwhisk/openwhisk.git
+cd openwhisk/tools/vagrant/custom
+
+```
+
+### Select one type of data store when creating vm
+Follow instructions [tools/db/README.md](../db/README.md) on how to configure a a datastore for OpenWhisk
+
+```
+# Provide credentials for couchdb database with admin permissions
+OW_DB=couchdb OW_DB_USERNAME=xxxxx OW_DB_PASSWORD=yyyyyy OW_DB_PROTOCOL=http OW_DB_HOST=1.2.3.4 OW_DB_PORT=5984 vagrant up
+
+# Provide credentials for cloudant database with admin permissions
+OW_DB=cloudant OW_DB_USERNAME=xxxxx OW_DB_PASSWORD=yyyyyy vagrant up
+```
+
+To try out OpenWhisk without managing your own CouchDB installation, you can start a CouchDB instance in a container as part of the OpenWhisk deployment. We advise that you use this method only as a temporary measure. Please note that:
+
+  * no data will persist between two creations of the container
+  * you will need to run the creation script every time you `clean` or `teardown` the system (see below)
+  * you will need to initialize the datastore each time (`tools/db/createImmportalDBs.sh`, see below)
+
+```
+# Configure with couchdb docker container running inside the vm 
+  vagrant up
+
+```
+
+### Wait for hello action output
+```
+wsk action invoke /whisk.system/samples/echo -p message hello --blocking --result
+{
+    "message": "hello"
+}
+```
+**Tip:** First time it takes 10 minutes or more depending on network speed, 
```

Comment:
fixed in https://github.com/openwhisk/openwhisk/pull/360/commits/3e3987983469b320ae5762d7ab60c5ecf7b6e747